test/lib_vnr/vnr_unit_tests/full/test_vnr_full.py

```diff
@@ -63,7 +63,7 @@ def test_vnr_full(target, tflite_model, vnr_conf):
     print("max_diff = ",np.max(np.abs(ref_output_double - dut_output_double)))
     arith_closeness, geo_closeness = test_utils.get_closeness_metric(ref_output_double, dut_output_double)
     print(f"arith_closeness = {arith_closeness}, geo_closeness = {geo_closeness}")
-    assert(geo_closeness > 0.98), "inference output geo_closeness below pass threshold"
+    assert(geo_closeness > 0.97), "inference output geo_closeness below pass threshold"
```

Comment:
test threshold creep! Do we know why this has changed?

Reply:
Nope, it happened straight after I switched the model, which was quite a lot of commits ago. Can try changing it back if you want

Reply:
I don't think I can run the old model anymore. I can try adjusting the threshold back, hoping it will pass, as there were quite a few changes

Reply:
yeah please

Reply:
@Allan-xmos both fail with this closeness
![image](https://github.com/user-attachments/assets/315af7c4-2e77-46dd-8472-23d2c8cf2bfb)